docs/Installation.md

```diff
@@ -29,7 +29,7 @@ Both directories are located at the root of the repository.
 ## Install Python (with Dependencies)
 
 In order to use ML-Agents, you need Python 3 along with
-the dependencies listed in the [requirements file](../python/requirements.txt).
+the dependencies listed in the [requirements file](../python/requirements.txt). You can install these dependencies using pip3 in the next step.  Additionally, you can download and install Python 3 from [here](https://www.python.org/downloads/).
```

Comment:
Not sure its a good idea to start discussing specific installations outside the subsections below. Please fold this info into both the Windows guide and the Mac/Unix users section below.

Reply:
ok i moved it down